packages/react/src/oxygen-ui-react-auth-components/SignIn/SignIn.tsx

```diff
@@ -21,35 +21,93 @@ import clsx from 'clsx';
+import SignInFooter from '../SignInFooter/SignInFooter';
+import SignInImage from '../SignInImage/SignInImage';
+import SignInLink, {SignInLinkProps} from '../SignInLink/SignInLink';
 import SignInPaper from '../SignInPaper/SignInPaper';
-import SignInTextField from '../SignInTextField/SignInTextField';
+import SignInPinInput from '../SignInPinInput/SignInPinInput';
+import SignInTextField, {SignInTextFieldProps} from '../SignInTextField/SignInTextField';
 import SignInTypography from '../SignInTypography/SignInTypography';
+import './sign-in.scss';
 
 export type SignInProps<C extends ElementType = ElementType> = {
   component?: C;
+  links?: SignInLinkProps[];
+  loginOptions?: SignInButtonProps[];
+  logo?: string;
+  subtitle?: string;
+  textFields?: SignInTextFieldProps[];
+  title?: string;
 } & Omit<BoxProps, 'component'>;
 
 type SignInCompoundProps = {
   Alert: typeof SignInAlert;
   Button: typeof SignInButton;
   Divider: typeof SignInDivider;
+  Footer: typeof SignInFooter;
+  Image: typeof SignInImage;
   Link: typeof SignInLink;
   Paper: typeof SignInPaper;
+  PinInput: typeof SignInPinInput;
   TextField: typeof SignInTextField;
   Typography: typeof SignInTypography;
 };
 
 const COMPONENT_NAME: string = 'SignIn';
 
+const renderTextFields = (textFields: SignInTextFieldProps[]): ReactElement[] =>
+  textFields.map((textFieldProps: SignInTextFieldProps, index: number) => (
+    <SignInTextField key={`sign-in-text-field${index + 1}`} {...textFieldProps} />
+  ));
+
+const renderLinks = (links: SignInLinkProps[]): ReactElement[] =>
+  links.map((linkProps: SignInLinkProps, index: number) => (
+    <div key={`sign-in-link-holder${index + 1}`}>
+      <SignInLink {...linkProps} />
+      <br />
+    </div>
+  ));
+
+const renderLoginOptions = (loginOptions: SignInButtonProps[]): ReactElement[] =>
+  loginOptions.map((loginOptionProps: SignInButtonProps, index: number) => (
+    <SignInButton key={`sign-in-button-social${index + 1}`} social {...loginOptionProps} />
+  ));
+
 const SignIn: ForwardRefExoticComponent<SignInProps> & WithWrapperProps & SignInCompoundProps = forwardRef(
   <C extends ElementType>(props: SignInProps<C>, ref: MutableRefObject<HTMLHRElement>): ReactElement => {
-    const {className, ...rest} = props;
+    const {className, title, subtitle, textFields, links, loginOptions, logo, ...rest} = props;
 
     const classes: string = clsx(`Oxygen${COMPONENT_NAME}`, className);
 
+    if (title || subtitle || textFields || links || loginOptions || logo) {
+      return (
+        <Box ref={ref} className={classes} {...rest}>
+          {logo && <SignInImage src={logo} />}
+
+          <SignInPaper>
+            {title && <SignInTypography title>{title}</SignInTypography>}
+
+            {subtitle && <SignInTypography subtitle>{subtitle}</SignInTypography>}
+
+            {textFields && renderTextFields(textFields)}
+
+            <SignInButton>Sign In</SignInButton>
```

Comment:
Shall we allow the user to customize this text as well?

Reply:
added with 312befac6d4c64ee27cdbf2819cfaff3e0a73cc0